### PR TITLE
[OpenVINO-EP] Update OpenVINO-EP documentation

### PR DESCRIPTION
**Pull request by MaajidKhan**
**Description**:  update the OpenVINO-EP docx

**Motivation and Context**
- Remove unwanted information
- clean up samples information
- Modify the expired links
- clean up the parts of docx that were wrongly placed while transitioning the pages to .html

### CODE REVIEW

**Review by oliviajain — APPROVED** [review]

File: docs/how-to/build/eps.md [hunk 1]
```diff
@@ -261,7 +261,7 @@ See more information on the OpenVINO Execution Provider [here](../reference/exec
 ```
 
 * `--use_openvino` builds the OpenVINO Execution Provider in ONNX Runtime.
-* `<hardware_option>`: Specifies the default hardware target for building OpenVINO Execution Provider. This can be overriden dynamically at runtime with another option (refer to [OpenVINO-ExecutionProvider.md](../reference/execution-providers/OpenVINO-ExecutionProvider.md) for more details on dynamic device selection). Below are the options for different Intel target devices.
+* `<hardware_option>`: Specifies the default hardware target for building OpenVINO Execution Provider. This can be overriden dynamically at runtime with another option (refer to [OpenVINO-ExecutionProvider](http://www.onnxruntime.ai/docs/reference/execution-providers/OpenVINO-ExecutionProvider.html#summary-of-options) for more details on dynamic device selection). Below are the options for different Intel target devices.
```
Comment (faxu) [inline docs/how-to/build/eps.md:264]: please use relative links. ../../reference/execution-providers/OpenVINO-ExecutionProvider.html#summary-of-options

File: docs/how-to/build/eps.md [hunk 2]
```diff
@@ -271,20 +271,21 @@ See more information on the OpenVINO Execution Provider [here](../reference/exec
...
+A minimum of two device's should be specified for a valid HETERO or Multi-Device Build.
 
-Example:
-HETERO:MYRIAD,CPU  HETERO:HDDL,GPU,CPU  MULTI:MYRIAD,GPU,CPU
+```
+Example's: HETERO:MYRIAD,CPU or HETERO:HDDL,GPU,CPU or MULTI:MYRIAD,GPU,CPU
+```
 
-For more information on OpenVINO Execution Provider&#39;s ONNX Layer support, Topology support, and Intel hardware enabled, please refer to the document [OpenVINO-ExecutionProvider.md](../reference/execution-providers/OpenVINO-ExecutionProvider.md) in <code>$onnxruntime_root/docs/execution_providers</code>
+For more information on OpenVINO Execution Provider&#39;s ONNX Layer support, Topology support, and Intel hardware enabled, please refer to the document [OpenVINO-ExecutionProvider](http://www.onnxruntime.ai/docs/reference/execution-providers/OpenVINO-ExecutionProvider.html)
```
Comment (faxu) [inline docs/how-to/build/eps.md:288]: please use relative links throughout for all updates

**Review by faxu — CHANGES_REQUESTED** [review]

File: docs/how-to/build/eps.md [hunk 3]
```diff
@@ -205,7 +205,7 @@ To build for Intel GPU, replace dnnl_opencl_root with the path of the Intel SDK
 
 ## OpenVINO
 
-See more information on the OpenVINO Execution Provider [here](../reference/execution-providers/OpenVINO-ExecutionProvider.md).
+See more information on the OpenVINO Execution Provider [here](http://www.onnxruntime.ai/docs/reference/execution-providers/OpenVINO-ExecutionProvider.html).
```
[the remark this answers is not in the document]
Comment (MaajidKhan) [reply docs/how-to/build/eps.md]: Done and verified

File: docs/how-to/build/eps.md (hunk 1 above)
Comment (MaajidKhan) [reply docs/how-to/build/eps.md]: Done and verified

File: docs/how-to/build/eps.md (hunk 2 above)
Comment (MaajidKhan) [reply docs/how-to/build/eps.md]: Done and verified

File: docs/reference/execution-providers/OpenVINO-ExecutionProvider.md [hunk 4]
```diff
@@ -22,7 +22,7 @@ For build instructions, please see the [BUILD page](../../how-to/build/eps.md#op
 ## Usage
 **C#**
 
-To use csharp api for openvino execution provider create a custom nuget package. Follow the instructions [here](../../how-to/build.md##build-nuget-packages) to install prerequisites for nuget creation. Once prerequisites are installed follow the instructions to [build openvino](../../how-to/build.md#openvino) and add an extra flag `--build_nuget` to create nuget packages. Two nuget packages will be created Microsoft.ML.OnnxRuntime.Managed and Microsoft.ML.OnnxRuntime.Openvino.
+To use csharp api for openvino execution provider create a custom nuget package. Follow the instructions [here](http://www.onnxruntime.ai/docs/how-to/build/inferencing.html#build-nuget-packages) to install prerequisites for nuget creation. Once prerequisites are installed follow the instructions to [build openvino](http://www.onnxruntime.ai/docs/how-to/build/eps.html#openvino) and add an extra flag `--build_nuget` to create nuget packages. Two nuget packages will be created Microsoft.ML.OnnxRuntime.Managed and Microsoft.ML.OnnxRuntime.Openvino.
```
Comment (faxu) [inline docs/reference/execution-providers/OpenVINO-ExecutionProvider.md:25]: please fix all links in the edits to relative links

Comment (MaajidKhan) [reply docs/reference/execution-providers/OpenVINO-ExecutionProvider.md]: Done

**Review by faxu — APPROVED** [review]

**Pull request merged**